core/storage/statistics/gae_models.py

```diff
@@ -450,13 +567,34 @@ class ExplorationAnnotationsModel(base_models.BaseMapReduceBatchResultsModel):
 
     @classmethod
     def get_entity_id(cls, exploration_id, exploration_version):
+        """Gets entity_id for a batch model based on given exploration state.
+
+        Args:
+            exp_id: str. ID of the exploration currently being played.
+            exploration_version: int. Version of the exploration currently
+                being played.
+
+        Returns:
+            str. Returns entity_id for a new instance of this class.
+        """
         return '%s:%s' % (exploration_id, exploration_version)
 
     @classmethod
     def create(
             cls, exp_id, version, num_starts, num_completions,
             state_hit_counts):
-        """Creates a new ExplorationAnnotationsModel."""
+        """Creates a new ExplorationAnnotationsModel and
+        then writes it on the datastore.
+
+        Args:
+            exp_id: str. ID of the exploration currently being played.
+            version: int. Version of exploration.
+            num_starts: int. Number of students who started the exploration.
+            num_completions: int. Number of students who have completed
+                the exploration.
+            state_hit_counts: dict. It describes the number of hits
```

Comment:
Describes the number of hits*

Don't use - "It describes"

Reply:
okay fine.